addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/HashDisclosureScannerUnitTest.java

```diff
@@ -29,7 +29,7 @@
 import org.parosproxy.paros.network.HttpMalformedHeaderException;
 import org.parosproxy.paros.network.HttpMessage;
 
-/** Unit test for {@link HashDisclosureScanner}. */
+/** Unit test for {@link org.zaproxy.zap.extension.pscanrulesBeta.HashDisclosureScanner}. */
```

Comment:
Should not be needed (same comment for other tests).

Reply:
I guess that was part of Eclipse moving or renaming things, will tweak.